org/allPRs.ts

```diff
@@ -5,10 +5,10 @@ export const prSize = async () => {
     const excludedExtensions = ['.xcodeproj', '.xcassets', '.xcworkspace'];
 
     // Get all modified and added files (unique)
-    const changedFiles = [...new Set([
```

Comment:
I found the Set to be breaking changedFiles so that it was empty…

Also a file can’t be both modified and created so it’s reasonable to skip the Set.